env/staging/celery-deployment.yaml

```diff
@@ -50,6 +50,8 @@ spec:
               value: '$(BULK_SEND_TEST_SERVICE_ID)'
             - name: CELERY_CONCURRENCY
               value: '$(CELERY_CONCURRENCY)'
+            - name: CELERY_DELIVER_SMS_RATE_LIMIT
```

Comment:
Do we need this to be in the non SMS celery deployment?

Reply:
well, not really. at least, we DO for the base (prod) `celery-deployment`  but not for the staging / scratch patch. I do dislike how we have variables all over the place even when they-re not used, so I'll take it out 👍